net/v2ray/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/v2ray/v2ray-core 4.22.1 v
+name                v2ray
+categories          net security
+platforms           darwin
+license             MIT
+maintainers         {@casouri gmail.com:casouri} openmaintainer
+description         A proxy server for bypassing network restrictions
+
+long_description    V2Ray supports many protocols, including Socks, HTTP, Shadowsocks, \
+                    VMess, and more. And each protocol may have its own transport, such as TCP, \
+                    mKCP, WebSocket, etc. Better yet, V2Ray has built in obfuscation \
+                    to hide traffic in TLS, and can run in parallel with web servers.
+
+homepage            https://www.v2ray.com
+
+checksums           rmd160  09e3e77b18fe0414639b9e9abaa91eb87fd87ca9 \
+                    sha256  5189ad64757acdf7e0196231db0e0d505c8c7d237c734cb70b25820abe0572cc \
+                    size    6835294
+
+build {
+    system -W ${worksrcpath}/main "env CGO_ENABLED=0 go build -o ./v2ray -ldflags \"-s -w\""
+    system -W ${worksrcpath}/infra/control/main "env CGO_ENABLED=0 go build -o ./v2ctl -tags confonly -ldflags \"-s -w\""
+}
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/main/v2ray ${destroot}${prefix}/bin
+    xinstall -m 755 ${worksrcpath}/infra/control/main/v2ctl ${destroot}${prefix}/bin
+    xinstall -m 755 ${worksrcpath}//release/config/geosite.dat ${destroot}${prefix}/bin
+    xinstall -m 755 ${worksrcpath}//release/config/geoip.dat ${destroot}${prefix}/bin
+    xinstall -d ${destroot}${prefix}/etc/v2ray
+    xinstall -m 640 ${worksrcpath}/release/config/config.json ${destroot}${prefix}/etc/v2ray
+}
+
+# until #2365 is implemented, manually handle config file
+post-destroot {
+    file rename ${destroot}${prefix}/etc/v2ray/config.json ${destroot}${prefix}/etc/v2ray/config.json.example
+}
+
+post-activate {
+    if {![file exists ${prefix}/etc/v2ray/config.json]} {
+        file copy ${prefix}/etc/v2ray/config.json.example ${prefix}/etc/v2ray/config.json
+    }
+}
+
+startupitem.create          yes
+startupitem.executable      ${prefix}/bin/v2ray --config ${prefix}/etc/v2ray/config.json
+
+notes                       "
+                            The default configuration file is copied to ${prefix}/etc/v2ray/config.json.
+                            It is used by the startup item (service). You should edit or replace it.
+
+                            v2ray also provides a utility command v2ctl, it can generate uuid and certificate
+                            and do other stuff.
+                            "
```

Comment:
There is way too much indenting going on here. It does not make sense to align with the lines above in this case. I'd push them back to about 4 spaces out.

Reply:
Changed to 4 spaces. Btw how does this indent thing work in Tcl? Is there a convention for macports?

Reply:
The only convention is to follow to mode line, if the Portfile has it. But looking at other Portfiles gives you an idea of the formatting that is used. It has nothing to do with Tcl.

Reply:
For the hopefully final nitpick, the last line with only the double quote should not be indented at all.

Reply:
Fixed.